src/index.html

```diff
@@ -17,11 +17,452 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
As per the code style rules, stylesheets with a `.scss` extension should not be linked directly in HTML. You should compile them into CSS files and link those instead.